PathTracking/stanley_controller/stanley_controller.py

```diff
@@ -147,15 +162,14 @@ def main():
     ax = [0.0, 100.0, 100.0, 50.0, 60.0]
     ay = [0.0, 0.0, -30.0, -20.0, 0.0]
 
-    cx, cy, cyaw, ck, s = cubic_spline_planner.calc_spline_course(
-        ax, ay, ds=0.1)
+    cx, cy, cyaw, _, _ = cubic_spline_planner.calc_spline_course(ax, ay, ds=0.1)
 
-    target_speed = 30.0 / 3.6  # [m/s]
+    target_speed = kph2mps(30.0)  # [m/s]
 
     max_simulation_time = 100.0
 
     # Initial state
-    state = State(x=-0.0, y=5.0, yaw=np.radians(20.0), v=0.0)
+    state = State(x=-0.0, y=5.0, yaw=np.deg2rad(20.0), v=0.0)
```

Comment:
@wlbksy One question. why are you using deg2rad instead of radians?

Reply:
They're equivalent. However, `deg2rad` is introduced since 1.13 version of `numpy`, its name is more clear than `radians`.

Reply:
oh. I see. You are right. We should use deg2rad.